modules/check_in/app/serializers/check_in/vaos/appointment_serializer.rb

```diff
@@ -10,6 +10,29 @@ class AppointmentSerializer
 
       attributes :kind, :status, :serviceType, :locationId, :clinic, :start, :end, :minutesDuration
 
+      attribute :telehealth do |object|
+        {
+          vvsKind: object&.telehealth&.dig('vvsKind'),
+          atlas: object&.telehealth&.dig('atlas')
+        }
+      end
+
+      attribute :extension do |object|
+        {
+          preCheckinAllowed: object&.extension&.dig('preCheckinAllowed'),
```

Comment:
same comment as above

Reply:
Addressed in [78a3200](https://github.com/department-of-veterans-affairs/vets-api/pull/18434/commits/78a320091fca2685bce7349f89a4fab0d2f6ebf8)